firefox_puppeteer/tests/test_windows.py

```diff
@@ -128,9 +128,16 @@ def closer(window):
         win2 = BaseWindow(lambda: self.marionette, win2.handle)
 
         self.assertEquals(len(self.marionette.chrome_window_handles), 2)
-
         win2.close(callback=closer)
-        win1.switch_to()
+
+        win1.focus()
+
+        # We cannot open a chrome window in the background yet, so this will fail
```

Comment:
I don't understand the comment. The opener script opens in the foreground, how does that mean we "cannot" open a window in the background?